### DISCUSSION TITLE
How to train a model for many types of elements

### DISCUSSION THREAD

**Discussion by swang066** [deepmodeling/deepmd-kit#1295] [Q&A]
If I have a system which has 20 different elements, how to train a model for them? Can I run AIMD for each pair of them and then use Deepmd-kit to combine them and train a model? Thank you very much!

**Comment by AnguseZhang** [comment]
See descrption for `type_map.raw` and `type.raw` in manual.
Btw, curretnly no one has trained a model with 20 different elements since its costs for GPU memory and training time is too much. What't the reason for training a system wtih 20 elements?

**Comment by njzjz** [comment]
If you only train each pair of them but not the mixture of them, the model can only be able to predict these pairs. However, in this case, I can't see the advantage of a single model instead of multiple models.
